File: importer-service/src/main/java/com/ems/importerservice/config/AppConfig.java
package com.ems.importerservice.config;

import org.springframework.boot.context.properties.EnableConfigurationProperties;
import org.springframework.context.annotation.Configuration;
import org.springframework.data.mongodb.config.EnableMongoAuditing;
import org.springframework.scheduling.annotation.EnableScheduling;

@Configuration
@EnableScheduling
@EnableMongoAuditing
@EnableConfigurationProperties({EventServiceClientProperties.class, ImportProperties.class})
public class AppConfig {
}

File: notification-service/src/main/kotlin/com/ems/notificationservice/config/AppConfig.kt
package com.ems.notificationservice.config

import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.context.annotation.Configuration
import org.springframework.data.mongodb.config.EnableMongoAuditing

@Configuration
@EnableMongoAuditing
@EnableConfigurationProperties(KafkaTopicsProperties::class, NotificationDeliveryProperties::class)
class AppConfig
